src/services/formatting/formatting.ts

```diff
@@ -445,7 +445,7 @@ namespace ts.formatting {
                     if ((<MethodDeclaration>node).asteriskToken) {
                         return SyntaxKind.AsteriskToken;
                     }
-                    // fall-through
+                // fall-through

```

Comment:
technically it is a leading comment for case clause, not a trailing comment for a block. Current behavior is preferable for cases like
```ts
switch(a) {
    // handling Value.A 
    case Values.A:
         break;
    // handling Value.B 
    case Values.B:
         break;
    // handling Value.C 
    case Values.C:
         break;
         
}
``` 

Here I definitely don't want comments for cases to be aligned with statements in previous case clause